save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/i18next/locales/Translations.kt

```diff
@@ -8,7 +8,8 @@ const val VULN_COLLECTION_RU = """
   "introMd": "|## Международный Архив Уязвимостей\n    |Данная страница предоставляет список публично раскрытых уязвимостей и инцидентов в области информационной безопасности.",
   "addNewMd": "|### Новые уязвимости\n    |Вы можете предложить свою [новую уязвимость](/vuln/create-vulnerability), если ее нет в нашем списке.\n    |После подтверждения со стороны администрации, она появится в архиве под уникальным ID.",
   "howToMd": "|### Как добавить уязвимость в проект?\n    |1. Создайте saveourtool [организацию](/create-organization) и [проект](/create-project);\n    |2. Откройте вкладку security вашего проекта;\n    |3. Создайте новую проблему и добавьте ее уникальный ID;\n    |\n    |В случае любых ошибок не стесняйтесь [связываться с нами](https://github.com/saveourtool/save-cloud/issues/new).",
-  "topRatingMd": "|### Рейтинговая система\n    |За каждую принятую уязвимость вы получаете очки рейтинга. Здесь вы можете ознакомиться с [рейтингом](/vuln/top-rating) пользователей и организаций."
+  "topRatingMd": "|### Рейтинговая система\n    |За каждую принятую уязвимость вы получаете очки рейтинга. Здесь вы можете ознакомиться с [рейтингом](/vuln/top-rating) пользователей и организаций.",
+  "Upload COSV files": "Загрузить COSV файлы"
```

Comment:
Why is it in `vulnerability-collection`? It is not VulnerabilityCollectionView

Reply:
I did not want to create new sections for the sake of one line